OsLogin/synth.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""This script is used to synthesize generated parts of this library."""
+
+import synthtool as s
+import synthtool.gcp as gcp
+import logging
+
+logging.basicConfig(level=logging.DEBUG)
+
+gapic = gcp.GAPICGenerator()
+common = gcp.CommonTemplates()
+
+v1beta = gapic.php_library(
+    service='oslogin',
+    version='v1beta',
+    artman_output_name='google-cloud-os-login-v1beta')
+
+v1 = gapic.php_library(
+    service='oslogin',
+    version='v1',
+    artman_output_name='google-cloud-os-login-v1')
+
+# copy all src
+s.move(v1beta / f'src/V1beta')
+s.move(v1 / f'src/V1')
+
+# copy proto files to src also
+s.move(v1beta / f'proto/src/Google/Cloud/OsLogin', f'src/')
+s.move(v1 / f'proto/src/Google/Cloud/OsLogin', f'src/')
+
+s.move(v1beta / f'tests/')
+s.move(v1 / f'tests/')
+
+# copy GPBMetadata file to metadata
+s.move(v1beta / f'proto/src/GPBMetadata/Google/Cloud/Oslogin', f'metadata/')
+s.move(v1 / f'proto/src/GPBMetadata/Google/Cloud/Oslogin', f'metadata/')
+
+# Remove @experimental tags from the V1 code
+s.replace(
+    'src/V1/**/*.php',
+    r'^(\s+\*\n)?\s+\*\s@experimental\n',
+    ''
+)
+
+# fix year
+s.replace(
+    'src/V1beta/Gapic/*GapicClient.php',
+    r'Copyright \d{4}',
+    r'Copyright 2017')
+s.replace(
+    'src/V1beta/OsLoginServiceClient.php',
+    r'Copyright \d{4}',
+    r'Copyright 2017')
+s.replace(
+    'tests/**/V1beta/*Test.php',
+    r'Copyright \d{4}',
+    r'Copyright 2018')
+s.replace(
+    'metadata/Common/*.php',
```

Comment:
Please correct me if I have got this wrong, but I don't think the metadata files have a copyright (maybe they should?)

Reply:
You are right, removed